ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -78,6 +78,11 @@ sporkaddr={{ dashd_sporkaddr }}
 sporkkey={{ dashd_sporkkey }}
 {% endif %}
 
+{% if dashd_walletnode is not none and coinjoin_wallet_count is defined %}
+coinjoinmultisession=1
+coinjoinsessions=16
```

Comment:
Maximum is 10, see https://docs.dash.org/projects/core/en/stable/docs/dashcore/wallet-arguments-and-commands-dashd.html#coinjoin-options